clang-format-apply.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Can the formatting scripts be moved into the `scripts/clang-format/` folder? Lets not clutter the project root 😄